en/docs/genai/getting-started/build-a-sample-hotel-booking-agent.md

```
@@ -1,21 +1,401 @@
+
+To use a different LLM instead, see [Model providers](/docs/genai/develop/components/model-providers) for the full list of supported providers (OpenAI, Azure OpenAI, Anthropic, and others).
+
+:::
+
+## Step 3: Configure the agent
+
+1. Select the **AI Agent** node in the design canvas.
+2. Set **Role** to `HotelBookingAssistant`.
+3. Set **Instructions** to:
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)